ChangeLog.d/8799.txt

```diff
@@ -0,0 +1,4 @@
+Bugfix
+   * mbedtls_pem_read_buffer() now performs a check on the padding data of
+     decrypted keys and it rejects invalid ones. It also parses and validates
+     the main ASN.1 SEQUENCE header.
```

Comment:
I think the sentence about the ASN1 header is too much detail: this isn't something we want to commit to. Also, it's partly preexisting, so wouldn't need to be in the changelog anyway.

(Also we might not make this change at all?)